content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md

```diff
@@ -59,25 +59,25 @@ Provide a name for the Aruba GreenLake SSO service (Okta application)
 
     **NameID = user.email**
 
-    **gl\_first\_name = user.FirstName**
+    **gl_first_name = user.FirstName**
```

Comment:
Some characters (underscore, hash, backslash) have special meanings in the Markdown syntax. If you want to use these special characters in a text, you have to escape them. The way to escape a special character is to add a backslash (\\) before it, for example: I do not want \_italic text\_ here.

Therefore, throughout the blog post, make sure to use the backslash in front of the "underscore" character for the variables such as gl_first_name.

You can check the preview of the updated blog post here:
https://deploy-preview-2421--hpe-dev-portal.netlify.app/blog/okta-sso-integration-for-green-lake-and-aruba-central/